src/lib/remark-plugins/rewrite-tutorial-links/index.ts

```diff
@@ -66,32 +66,40 @@ export const rewriteTutorialLinksPlugin: Plugin = () => {
 }
 
 function handleRewriteTutorialsLink(node: Link | Definition) {
+  node.url = rewriteTutorialsLink(node.url, TUTORIAL_MAP)
+}
+
+export function rewriteTutorialsLink(
```

Comment:
This is a very light refactor to expose a `rewriteTutorialsLink` function that we can re-use elsewhere. In brief:
- changes `node.url` to new `let newUrl`, which is returned at the end of the function
- adds an argument for `TUTORIAL_MAP`, so that it can be provided without having to call the remark plugin
- everything else within `rewriteTutorialsLink` stays the same
- wraps `rewriteTutorialsLink` in `handleRewriteTutorialsLink` to keep the same clean call as above in each `visit` call